app/components/Input/Input.tsx

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from 'react';
+import styles from './Input.module.scss';
+import Image from 'next/image';
+
+
+export default () => {
+    const [showIcon, setShowIcon] = useState(true);
+
+    const showing = () => {
+        setShowIcon(!showIcon);
+    }
+
+    return(
+        <div className={styles.container} id={styles.c} onFocus={showing} onBlur={showing}>
+            { showIcon && <Image src={'./searchIconBlur.svg'} alt='image' width={24} height={24} className={styles.img} />}
```

Comment:
dont give image component  in src path give only name of icon or image  , other logic write in image component